src/NuGet.Core/NuGet.Packaging/ContentModel/ContentItem.cs

```diff
@@ -7,12 +7,25 @@ namespace NuGet.ContentModel
 {
     public class ContentItem
     {
+        private Dictionary<string, object> _properties;
         public string Path { get; set; }
-        public IDictionary<string, object> Properties { get; } = new Dictionary<string, object>();
+
+        public Dictionary<string, object> Properties
+        {
+            get => _properties ?? CreateDictionary();
```

Comment:
@halter73 told me off for doing that once; now is habit 😄 

Smaller for inline without the new; though unlikely to be significant.